Makefile

```diff
@@ -22,13 +25,22 @@ quality-js:
 
 deps:
 	pip install -r requirements.txt
-	bower install
 
 deps-test:
 	pip install -r test_requirements.txt
+	bower install
 
 tools:
-	npm install "eslint@^2.12.0" eslint-config-edx "eslint-plugin-dollar-sign@0.0.5" "eslint-plugin-import@^1.9.2"
+	npm install bower "eslint@^2.12.0" eslint-config-edx "eslint-plugin-dollar-sign@0.0.5" "eslint-plugin-import@^1.9.2"
+
+send_coveralls:
+	coveralls-lcov -v -n video_xblock/static/coverage/PhantomJS\ 2.1.1\ \(Linux\ 0.0.0\)/lcov.info > coverage.json
```

Comment:
Is it possible to enquote `.info` file name to avoid backslashes? I.e. `".../PhantomJS 2.1.1 .../lcov.info"`